robottelo/cli/activationkey.py

```diff
@@ -81,15 +75,3 @@ def remove_repository(cls, options=None):
         """Disassociate a resource"""
         cls.command_sub = 'remove-repository'
         return cls.execute(cls._construct_command(options))
-
-    @classmethod
-    def remove_subscription(cls, options=None):
-        """Remove subscription"""
-        cls.command_sub = 'remove-subscription'
-        return cls.execute(cls._construct_command(options))
-
-    @classmethod
-    def subscriptions(cls, options=None, output_format=None):
```

Comment:
@jeremylenz we'll have to remove their usage across Robottelo. For example, this is still used in https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_host.py#L1852
Same for other removals of the cli options/commands, they might still be used in some tests. 

Also, the api endpoints referred [here](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/endtoend/test_api_endtoend.py#L46) should be removed too.